ios/RNCallKeep/RNCallKeep.m

```diff
@@ -55,7 +56,7 @@ - (instancetype)init
 #endif
     if (self = [super init]) {
         _isStartCallActionEventListenerAdded = NO;
-        _delayedEvents = [NSMutableArray array];
+        if (_delayedEvents == nil) _delayedEvents = [NSMutableArray array];
```

Comment:
`init` is called repeatedly, deleting any events queued up here. This avoids that.